src/qengine/opencl.cpp

```diff
@@ -1923,12 +1926,12 @@ QInterfacePtr QEngineOCL::Clone()
     QEngineOCLPtr copyPtr = std::make_shared<QEngineOCL>(
         qubitCount, 0, rand_generator, complex(ONE_R1, ZERO_R1), doNormalize, randGlobalPhase, useHostRam, deviceID);
 
-	copyPtr->clFinish();
+    copyPtr->clFinish();
 
-	copyPtr->runningNorm = runningNorm;
-	//WAIT_COPY(*stateBuffer, *(copyPtr->stateBuffer), sizeof(complex) * maxQPower);
+    copyPtr->runningNorm = runningNorm;
+    // WAIT_COPY(*stateBuffer, *(copyPtr->stateBuffer), sizeof(complex) * maxQPower);
```

Comment:
Should probably remove these legacy bits.

Reply:
Done.